chain-core/BUILD-WASM.md

```diff
@@ -0,0 +1,83 @@
+WASM BUILD GUIDE
+----------------------------
+
+## install emcc for compile WASM
+1. use ubuntu 18.x
+2. git clone https://github.com/emscripten-core/emsdk.git
+3. cd emsdk
+4. ./emsdk install latest
+5. ./emsdk activate latest
+6. source ./emsdk_env.sh
+
```

Comment:
for macOS? `brew install emscripten`?

Reply:
for mac, i'll try

Reply:
mac working fine, added to markdown for additional setup